cmd/mt-whisper-importer-writer/verify.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+
+# this is a tool to verify if the data imported via the whisper importer is
+# actually correct. it uses two inputs, the output of whisper-dump.py and the
+# output of mt-store-cat.
+#
+# pipe the whisper-dump.py output into a file:
+# $> whisper-dump.py storage/whisper/some/id/of/a/metric/1.wsp  > /tmp/whisper_dump
+#
+# pipe the mt-store-cat output into another file:
+# $> mt-store-cat -print-ts -from 1487764800 -to 1487785801 normal id 1.d588ebb28e4d2ca64d828cb1eb6066d4 5h > /tmp/mt_store_dump
```

Comment:
also have you tried introducing some mistakes (maybe a missing value on top, one at the end, a changed value in the middle etc) to see that verify.py works correctly?